api/filter_dimension_options.go

```diff
@@ -498,7 +498,15 @@ func (api *FilterAPI) patchFilterBlueprintDimensionHandler(w http.ResponseWriter
 			http.Error(w, err.Error(), http.StatusBadRequest)
 			return
 		}
-		totalValues += len(patch.Value)
+		v2, ok := patch.Value.([]interface{})
```

Comment:
I think here we should call `getStringArrayFromInterface` in order to guarantee that the values are correct, and `getStringArrayFromInterface` should return an error if any value is incorrect.
Otherwise, we might start applying patches with some invalid input.
```
v2, err := getStringArrayFromInterface(patch.Value)
if err != nil {
...
```

Reply:
Thanks for review.
Improvement applied.